google_analytics/tests/test_ga.py

```diff
@@ -61,6 +61,50 @@ def test_ga_url(self):
         self.assertEqual(parse_qs(ga_url1).get('tid'), ['ua-test-id'])
         self.assertEqual(parse_qs(ga_url1).get('uip'), ['100.100.200.10'])
 
+    def test_uip_for_empty_http_header(self):
+        client = Client(HTTP_X_FORWARDED_FOR='')
+        response = client.get(
+            '/google-analytics/?p=%2Fhome&utmdebug='
+            'True&r=test.com&tracking_code=ua-test-id')
+        ga_url1 = response.get('X-GA-MOBILE-URL')
+        self.assertIsNone(parse_qs(ga_url1).get('uip'))
```

Comment:
Shouldn't we fall back to the `REMOTE_ADDR` if the header is present but empty?

Reply:
Yes, but I didn't specify either. Let me add a test for that specifically.